src/prefect/orion/models/work_queues.py

```diff
@@ -62,7 +62,7 @@ async def create_work_queue(
         ).where(db.WorkQueue.work_pool_id == data["work_pool_id"])
         priority = (await session.execute(max_priority_query)).scalar()
 
-        model = db.WorkQueue(**data, priority=priority)
+        model = db.WorkQueue(**data, priority=priority + 1)
```

Comment:
Why + 1?

Reply:
This will make sure that it's lowest in priority when it's created. Otherwise the queue will have the same priority as the lowest priority existing work queue in the pool.

Reply:
This seems worth some sort of comment before this section — the lack of documentation in these `if` blocks makes this all pretty inscrutable as far as its intentions go.

Reply:
Yup, I added some comments and cleaned up the logic.

Reply:
Thanks!